.github/workflows/phase_3_yocto.yml

```diff
@@ -0,0 +1,193 @@
+---
+name: Phase 3 - Yocto
+on:
+  push:
+    paths:
+      - .github/workflows/phase_3_yocto.yml
+
+env:
+  YOCTO_TAG: styhead
+  PARLAY_VERSION: 0.6.0
+  SBOMASM_VERSION: 0.1.5
+  SBOMQS_VERSION: 0.1.9
+  TRIVY_VERSION: 0.54.1
+
+jobs:
+  Generate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Environment
+        run: |
+          sudo apt update
+          sudo apt install -y gawk wget git diffstat unzip texinfo gcc build-essential chrpath socat cpio python3 python3-pip python3-pexpect xz-utils debianutils iputils-ping python3-git python3-jinja2 python3-subunit zstd liblz4-tool file locales libacl1
```

Comment:
add python3-websockets

Reply:
I had to do a pip install. The version of websockets in apt is apparently to low. Here is the warning I got:
```
WARNING: BB_HASHSERVE_UPSTREAM is not valid, unable to connect hash equivalence server at 'wss://hashserv.yoctoproject.org/ws': ImportError('Websockets version 9.1 is less than minimum required version 10.0')
```